### ISSUE TITLE
xlsm format not working

### ISSUE THREAD

**Issue by bouaik** [weshatheleopard/rubyXL#425]
I'm generating a xlsm format but office 365 excel can't open it

```ruby
workbook = RubyXL::Workbook.new

worksheet.add_cell(0, 0, 'Test')      # Sets cell A1 to string "Test"

workbook.write("file.xlsm")
```

**Comment by harsh183** [comment]
Could you generate an xlsx and then convert that to xlsm?

**Comment by bouaik** [comment]
convert it directly in excel you mean ??

**Comment by harsh183** [comment]
Yeah, like can you generate an xlsx file using rubyXL and then convert to xlsm using an external tool?

**Comment by rogsmith** [comment]
@bouaik @harsh183 did you ever find a solution to this? I am running into same issue with reading an xlsm file and then writing it back out.

**Comment by harsh183** [comment]
I didn't run into this myself since I use xlsx files and they work great for me. Unless transforming to an xlsx file is an option for you, I'm not sure I have any other tips

**Comment by rogsmith** [comment]
thx @harsh183 yeah if I convert to xlsx then I will lose the macros which I need to run, seems like only other option is re-werite my stuff in python on windows I thought based on code it should work with writing .xlsm

**Comment by weshatheleopard** [comment]
Guys, I need a sample of the file you are having problems with. I cannot operate purely on hearsay.

**Comment by rogsmith** [comment]
@weshatheleopard here is a simple file I am just opening it and then immediately saving. and it has errors when opening. I have attached both input and output.

https://we.tl/t-EPno3tCYsj
